Harden-Windows-Security.ps1

```diff
@@ -66,8 +66,11 @@ Version 2023.1.17: fixed text spacing and colors to improve readability, removed
 ## 
 Version 2023.1.22: Added a notice at the beginning of the script to remind the user to read GitHub readme page. added Smart App Control to the Windows Security (Defender) section. script asks for confirmation before turning on Smart App Control.
 ## 
-Version 2023.1.23: Changed the registry modification function with a more advanced one. changed code style to reduce function call and use hash tables. Improved the overall design and style of the script. Changed 'ConsentPromptBehaviorAdmin' from UAC category to a #TopSecurity tagged command. see this GitHub issue for more info. https://github.com/HotCakeX/Harden-Windows-Security/issues/2#issuecomment-1400115303
-
+Version 2023.1.23: Changed the registry modification function with a more advanced one. changed code style to reduce function call and use hash tables. Improved the overall design and style of the script. Changed 'ConsentPromptBehaviorAdmin' from UAC category to a #TopSecurity tagged command. see this GitHub issue for more info. https://github.com/HotCakeX/Harden-Windows-Security/issues/2#issuecomment-1400115303 . removed PowerShell Core requirement.
+## 
+Version 2023.1.24: enforce encryption type on removable and fixed drive types to full disk encryption instead of only used disk encryption
+## 
+Version 2023.1.25: The script now applies the official Microsoft Security Baselines and on top of that applies as many of the script settings as possible using Group Policy, the rest of the settings that aren't possible to be applied using Group Policy continue to be applied using registry and PowerShell Cmdlets.

```

Comment:
What happens if this script is executed on Windows 10 Home? It doesn't have Group Policy, right?

Reply:
Hi,
yes, neither Group Policy not Bitlocker is available on Home editions.
https://www.microsoft.com/en-us/windows/compare-windows-10-home-vs-pro

in my opinion, Windows home edition is for usage in safe environments where Windows out-of-the-box will take care of the security threats. the kind a worst-case scenarios that this script prepares the system for isn't possible on Home editions.

I'll test it on Home edition VM soon to see what type of errors I'll get